integration-cli/docker_utils.go

```diff
@@ -679,6 +679,22 @@ func dockerCmdInDirWithTimeout(timeout time.Duration, path string, args ...strin
 	return out, status, err
 }
 
+func findContainerExitCode(c *check.C, name string, vargs ...string) (string, int) {
+	args := append(vargs, "inspect", "--format='{{ .State.ExitCode }} {{ .State.Error }}'", name)
+	cmd := exec.Command(dockerBinary, args...)
+	out, _, err := runCommandWithOutput(cmd)
+	if err != nil {
+		c.Fatal(err, out)
+	}
+	// return 127 for Windows build testing
```

Comment:
Just to be clear - this is running from a Windows CLI against a Linux daemon when you say "In Windows...." and later "when running in Windows".

If so, the exit code is set by the daemon, so there shouldn't be a reason you have to do client-specific processing.